components/identities/actions/transferFormModal.tsx

```diff
@@ -16,14 +16,18 @@ import { StarknetIdJsContext } from "../../../context/StarknetIdJsProvider";
 import ConfirmationTx from "../../UI/confirmationTx";
 import { useNotificationManager } from "../../../hooks/useNotificationManager";
 import { NotificationType, TransactionType } from "../../../utils/constants";
+import { Identity } from "../../../utils/apiWrappers/identity";
+import { transfer } from "../../../utils/callData/identityChangeCalls";
```

Comment:
We want to keep the same way of importing `registerCalls` for example. 

Would be like `identityChangeCalls.transfer(...)`. It's useful to be sure what you're using in the code directly without having ton to ctrl + click ...